pkg/api/converterfromapi.go

```diff
@@ -735,6 +735,7 @@ func convertKubernetesConfigToVLabs(api *KubernetesConfig, vlabs *vlabs.Kubernet
 	vlabs.CustomCcmImage = api.CustomCcmImage
 	vlabs.UseCloudControllerManager = api.UseCloudControllerManager
 	vlabs.CustomWindowsPackageURL = api.CustomWindowsPackageURL
+	vlabs.WindowsNodeBinariesURL = api.WindowsNodeBinariesURL
```

Comment:
Let's also update the corresponding unit tests to reflect these new fields.

Reply:
There do not appear to be any unit tests for `convertfromapi.go`